### PR TITLE
fix: upgrade dockerhub url v2 to check for tags

### PR DESCRIPTION
**Pull request by tiagolobocastro**
This allows us to avoid re-publishing release tags "proper". Move the check till after the build is done, so we can still push the branch/alias tags.

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

**Review by Ab-hishek — APPROVED** [review]

File: scripts/release.sh [hunk 1]
```diff
@@ -11,16 +11,17 @@ set -euo pipefail
 
 # Test if the image already exists in dockerhub
 dockerhub_tag_exists() {
-  curl --silent -f -lSL https://index.docker.io/v1/repositories/$1/tags/$2 1>/dev/null 2>&1
+  curl --silent -f -lSL https://hub.docker.com/v2/repositories/$1/tags/$2 1>/dev/null 2>&1
 }
 
 # Derives tag name from the git repo. That is git tag value or short commit
```
Comment (chriswldenyer) [inline scripts/release.sh]: This comment needs updating, I think

**Review by chriswldenyer — APPROVED** [review]
Looks reasonable

File: scripts/release.sh (hunk 1 above)
Comment (tiagolobocastro) [reply scripts/release.sh]: Good catch, done on this and other repos

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**